src/Lean/Server/Rpc/Basic.lean

```diff
@@ -19,12 +19,19 @@ first connect to the session using `$/lean/rpc/connect`. -/
 
 namespace Lean.Lsp
 
-/-- An object which RPC clients can refer to without marshalling. -/
+/--
+An object which RPC clients can refer to without marshalling.
+
+The language server may serve the same `RpcRef` multiple times and maintains a reference count
+to track how often it has served the reference.
+If clients want to release the object associated with an `RpcRef`,
+they must release the reference as often as they have received it from the server.
```

Comment:
```suggestion
A reference to an object in an `RpcObjectStore`.
Used to allow RPC clients to refer to data stored in the server.

The language server may serve the same `RpcRef` multiple times and maintains a reference count
to track how often it has served the reference.
If clients want to release the object associated with an `RpcRef`,
they must release the reference as many times as they have received it from the server.
```
(Afaik 'often' is about ongoing frequency rather than the amount of times something happens/has happened.)